.travis/opensuse/Dockerfile.deps.tmpl

```diff
@@ -23,8 +23,8 @@ RUN zypper install --no-confirm --no-recommends \
 	meson \
 	ninja \
 	pkgconf \
-	python-devel \
-	python-gobject \
+	python2-devel \
+	python2-gobject \
 	python2-six \
```

Comment:
```
No provider of 'python2-six' found.
No provider of 'python2-gobject' found.
```

Please also remove
```yaml
  allow_failures:
    ...
    - name: "openSUSE tumbleweed"
```
so that the CI will report accurate success or failure.

Reply:
Ugh I forgot that Python 2 stuff is just gone now... Let me fix that...